Resources/Prototypes/DeltaV/Entities/Clothing/Ears/headset_alt.yml

```diff
@@ -0,0 +1,14 @@
+- type: entity
+  parent: ClothingHeadsetAlt
+  id: ClothingHeadsetAltAdminAssistant
+  name: administrative assistant over-ear headset
+  components:
+  - type: ContainerFill
+    containers:
+      key_slots:
```

Comment:
worth considering, would they be more useful as assistants if they could access other channels? Or are we expecting heads of staff to hand out keys when needed?

Reply:
Going to leave it to departments to hand out keys when needed, since most of their responsibilities can be done with the command and common channels. If they really need the keys after live testing though, Ill PR them getting more keys.